pkg/trace/traceutil/azure.go

```diff
@@ -144,6 +156,12 @@ func getLinuxRuntime(getenv func(string) string) (rt string) {
 	return rt
 }
 
+func inFunctionApp() bool {
+	_, existsRuntime := os.LookupEnv("FUNCTIONS_WORKER_RUNTIME")
+	_, existsVersion := os.LookupEnv("FUNCTIONS_EXTENSION_VERSION")
+	return existsRuntime && existsVersion
+}
+
```

Comment:
There will be some small performance hits to lookup the env here, throw away the value, and then look it up again immediately after.